docker-plugin/src/main/java/com/nirima/jenkins/plugins/docker/DockerTemplate.java

```diff
@@ -102,6 +105,7 @@
 
     @DataBoundConstructor
     public DockerTemplate(String image, String labelString,
+    		              Integer numExecutors,
```

Comment:
I actually had `int` to start, saw there were some other recent fields that used `Integer` so I switched it at the last moment. Argument about not null makes sense to me, I'll change and fix indent tonight.